### ISSUE TITLE
Textarea scrollbar sometimes overlaps text

### ISSUE THREAD

**Issue by Desour** [minetest/minetest#8949]
##### Minetest version
```
Minetest 5.1.0-dev-94a5df7-dirty (Linux)
Using Irrlicht 1.8.4
BUILD_TYPE=Release
RUN_IN_PLACE=1
USE_GETTEXT=1
USE_SOUND=1
USE_CURL=1
USE_FREETYPE=1
USE_LUAJIT=1
STATIC_SHAREDIR="."
```

##### Summary
When writing to a texarea that has a scrollbar, some letters can be behind the scrollbar.
After reopening it's somehow fixed.
Example: (The word in the first line should be "tempor".)
![screenshot_20190918_194746](https://user-images.githubusercontent.com/7613443/65173218-a7db9280-da4e-11e9-8486-d9a64febc703.png)

##### Steps to reproduce
- Start a minetest_game world.
- Get a book and open it.
- Add a title and copy [this text](https://la.wikisource.org/wiki/Lorem_ipsum) into the textarea.
- Look at the words right at the scrollbar. You should be able to see something like in the screenshot above.
- Click on save and reopen. This fixes it.

**Comment by ClobberXD** [comment]
Long time bug; thanks for reporting it.

**Comment by random-geek** [comment]
This was fixed by #7816. I guess it un-fixed itself since then.
